x-pack/plugins/fleet/server/index.ts

```diff
@@ -39,6 +39,7 @@ export type { FleetConfigType } from './config';
 export type { PackagePolicyClient } from './services/package_policy_service';
 
 export { relativeDownloadUrlFromArtifact } from './services/artifacts/mappings';
+export { transformOutputToFullPolicyOutput } from './services/agent_policies/full_agent_policy';
```

Comment:
nit: Do you think it will make sense to expose this as part of the output client? it will maybe make it more obvious that it's consumed by other plugins